bsp/stm32/stm32l412-st-nucleo/board/Kconfig

```diff
@@ -0,0 +1,74 @@
+menu "Hardware Drivers Config"
+
+config SOC_STM32L412RB
+    bool
+    select SOC_SERIES_STM32L4
+    select RT_USING_COMPONENTS_INIT
+    select RT_USING_USER_MAIN
+    default y
+
+menu "Onboard Peripheral Drivers"
+	config BSP_USING_STLINK_TO_USART
```

Comment:
Please use space for indentation.

Reply:
I had used space instead of tab for indentation